pkg/apis/velero/v1/backup_storage_location.go

```diff
@@ -80,6 +80,11 @@ type BackupStorageLocationSpec struct {
 	// AccessMode defines the permissions for the backup storage location.
 	// +optional
 	AccessMode BackupStorageLocationAccessMode `json:"accessMode,omitempty"`
+
+	// BackupSyncPeriod defines how frequently to sync backup API objects from object storage.
```

Comment:
Probably worth mentioning that "0" means never.

Reply:
I missed this, added 0 means disabling sync.